book_source/02_basic_users_guide/01_getting_started.Rmd

```diff
@@ -4,34 +4,33 @@
 
-There are two ways of using PEcAn, via the web interface and directly within R. Even for users familiar with R, using the web interface is a good place to start because it provides a high level overview of the PEcAn workflow.The quickest way to get started is to download the virtual machine or use an AWS instance.
+There are two ways of using PEcAn, via the web interface and directly within R. Even for users familiar with R, using the web interface is a good place to start because it provides a high level overview of the PEcAn workflow. The quickest way to get started is to download the virtual machine or use an AWS instance.
 
-## Download the PEcAn VM
+## Installing and running PEcAn
 
-1. PEcAn consists of a set of scripts and code that is compiled within a Linux operating system and saved in a “virtual machine (VM)”. Virtual machines allow for running consistent set-ups without worrying about differences between operating systems, library dependencies, compiling the code, etc.
+### Using the Virtual Machine
 
-2. To run the PEcAn VM you will need to install VirtualBox, the program that runs the virtual machine [http://www.virtualbox.org](http://www.virtualbox.org). On Windows you may see a warning about Logo testing, it is okay to ignore the warning.
+PEcAn consists of a set of scripts and code that is compiled within a Linux operating system and saved in a "virtual machine" (VM). Virtual machines allow for running consistent set-ups without worrying about differences between operating systems, library dependencies, compiling the code, etc.
 
-3. After you have Virtual Box installed you’ll need to download the PEcAn virtual machine: [http://opensource.ncsa.illinois.edu/projects/artifacts.php?key=PECAN](http://opensource.ncsa.illinois.edu/projects/artifacts.php?key=PECAN). The virtual machine is available under the "**Files**" header. Click the 32 or 64 bit ".ova" file and note that the download is ~5 GB so will take from several minutes to hours depending on the connection speed.
+1. **Install VirtualBox** This is the software that runs the virtual machine. You can find the download link and instructions at [http://www.virtualbox.org](http://www.virtualbox.org). *NOTE: On Windows you may see a warning about Logo testing, it is okay to ignore the warning.*
 
-4. To open up the virtual machine you'll first want to open up VirtualBox.
+2. **Download the PEcAn VM** You can find the download link at [http://opensource.ncsa.illinois.edu/projects/artifacts.php?key=PECAN](http://opensource.ncsa.illinois.edu/projects/artifacts.php?key=PECAN), under the "**Files**" header. Click the 32 or 64 bit (recommended) ".ova" file to begin the download. Note that the file is ~5 GB, so this download can take several minutes to hours depending on your connection speed.
```

Comment:
there is no more 32 or 64 bit, only pecan.

Reply:
So, I can purge _all_ references to 32 vs. 64 bit VMs, etc., right?

Reply:
correct, there is only 64 bit now. No more port 3280/3222.

Reply:
See ea3bcbe77. It's a first pass at purging all references to 32 bit architecture. I'm not sure all the documentation is still correct, but then again, it was probably wrong anyway. I'll give this a more detailed look later.